apps/builder/app/canvas/shared/inert.ts

```diff
@@ -0,0 +1,32 @@
+let resetTimeoutHandle: number | undefined = undefined;
+
+const resetAutoDisposeInert = () => {
+  document.body.removeAttribute("inert");
+  clearTimeout(resetTimeoutHandle);
+  resetTimeoutHandle = undefined;
+};
+
+// 1000 ms is a reasonable time for the preview to reset.
+// Anyway should never happen after user has finished preview changes (can happen during preview changes)
+const AUTO_DISPOSE_INERT_TIMEOUT = 1000;
+
+// A brief delay to ensure mutation observers within the focus scope are activated by the preview changes.
```

Comment:
this could use an example, to better understand what can happen, e.g. user does x, then something happens, then we wait then .etc ...  you know a practical example, its very hard to understand what is happening there without full knowledge

Reply:
Example added at setInert